website/pages/docs/cli/commands/cloudquery_completion_bash.md

```diff
@@ -15,27 +14,27 @@ If it is not installed already, you can install it via your OS's package manager
 
 To load completions in your current shell session:
 
-```powershell
-    source <(cloudquery completion bash)
+```bash
+	source <(cloudquery completion bash)
```

Comment:
Also need an advice here

Default completion description doesn't have codeblock markups and it breaks Nextra mdx parser. With this we have to fix it manually after generation
https://github.com/spf13/cobra/blob/a0aadc68eb88af3acad9be0ec18c8b91438fd984/completions.go#L654

Not sure how to fix it  :)
Perhaps I can extract completion.go as described here